### DISCUSSION TITLE
求教：怎么让别人看不出来是用nuxt写的。Ask for advice: how to make others invisible is written with nuxt

### DISCUSSION THREAD

**Discussion by againF** [nuxt/nuxt#9644] [Questions (Nuxt 2)]
nuxt generate生成的网站里，会有一些nuxt标记，我知道可以更改nuxt.config.js里的
build: {
  publicPath: '/customName/'
}
来修改文件输出目录，但是HTML里还有一些nuxt-link,还有id="__nuxt"，还有window.__NUXT__这些蛛丝马迹可以看出是用nuxt写的，有什么办法可以把所有nuxt都换个名字，目的是让别人看不出这个网站是用nuxt写的。
In the website generated by nuxt generate, there will be some nuxt tags. I know that you can change the nuxt.config.js
build: {
publicPath:'/customName/'
}
To modify the file output directory, but there are some nuxt-links in the HTML, and id="__nuxt", and window.__NUXT__ These clues can be seen that they are written in nuxt. Is there any way to change all nuxt The name, the purpose is to let others not see that this website is written in nuxt.

**Comment by azataiot** [comment]
你为什不希望别人看出来是nuxt 写的呢？ 出于什么样的原因和目的？ 解决对应的问题可能比“不让别人知道是nuxt写的“可能会更加的简单。

Why don't you want others to know that your application is built with 'nuxt.js'?  Is there any reason, like a security issue or else? if yes, then you should to try to solve those problem, cause it seems to be much more logical and easy to do so than 'not let others know that the app is built with nuxt'

**Reply by againF** [reply]
我们领导担心有人研究出来了Nuxt的漏洞然后攻击我们的网站，所以想隐藏相关信息。

**Reply by azataiot** [reply]
> 我们领导担心有人研究出来了Nuxt的漏洞然后攻击我们的网站，所以想隐藏相关信息。

但是有一个很大的问题。nuxt 毕竟是前端的东西，你再怎么改，如果有人真的想攻击，他还是可以发现并攻击的。

所以安全内容还是得放到后端服务器里。启用nginx 的一些访问限制，对api 进行一些必要的验证。或者ip 地址黑白名单，客户端user agent 过滤等等。 

后端防护比前端更靠谱些。
